src/Adapters/Storage/Postgres/PostgresStorageAdapter.js

```diff
@@ -747,6 +747,33 @@ export class PostgresStorageAdapter {
       });
   }
 
+  schemaUpgrade(className, schema) {
+    debug('schemaUpgrade', { className, schema });
+
+    return this._client.tx('schemaUpgrade', t => {
+      return t.any('SELECT column_name FROM information_schema.columns WHERE table_name = $<className>', { className })
```

Comment:
Method [map](http://vitaly-t.github.io/pg-promise/Database.html#map) should be used here.